nbdime/diffing/notebooks.py

```diff
@@ -194,6 +195,8 @@ def diff_mime_bundle(a, b, path=None):
     "/cells/*": diff,
     "/cells/*/outputs": diff_sequence_multilevel,
     "/cells/*/outputs/*": diff_single_outputs,
+    "/cells/*/attachments": diff_sequence_multilevel,
+    "/cells/*/attachments/*": diff_mime_bundle,
```

Comment:
These guys will not work as they should since `attachments` is a dict of named bundles so:
 - `diff_sequence_multilevel` doesn't make sense
 - `"/cells/*/attachments/*"` will never match, as currently star only matches list indices (I think?)